css/select_a_meal.css

```diff
@@ -44,7 +44,9 @@
     color:#4e4e4e;
 
 }
-
+.lab-selected{
+    background:#4e4e4e !important;
```

Comment:
ah strange. Try removing it and see if it still works as intended. `!important` just means that this attribute will take precedence over any of the selectors that match. So for example
```CSS
body {
   background: white;
}

.lab {
    background: blue;
}
```
Means that the .lab background will override the body background since its more specific. Thats the cascading part int Cascading Style Sheet. With important though you can do this

```CSS
body {
    background: white !important;
}
.lab {
    background: blue;
}
```

And the `.lab` background will still be white because !important overrides the Cascading.